x-pack/plugins/observability/public/pages/slos/components/toggle_slo_view.tsx

```diff
@@ -46,11 +47,10 @@ export function ToggleSLOView({
   sloView,
   onChangeView,
   onToggleCompactView,
+  onStateChange,
   isCompact = true,
   sloList,
-  loading,
-  initialState,
-  onStateChange,
+  state,
```

Comment:
@dominiqueclarke We can pass one more prop here, for example `isGroupView`, and if it is a group view, let's hide the `"Showing {currentCount} of {total} {slos}"` info.